Framework/PythonInterface/plugins/CMakeLists.txt

```diff
@@ -136,6 +137,7 @@ set(PYTHON_PLUGINS
     algorithms/MuonMaxent.py
     algorithms/MuscatSofQW.py
     algorithms/NMoldyn4Interpolation.py
+    algorithms/NOMADMedianDetectorTest.py
```

Comment:
A test file here will confuse future developers.
It this is an actual test file, please move it to the correct section.
Otherwise, please considering renaming the file to something without the keyword test inside.